packages/Badge/src/index.tsx

```diff
@@ -36,10 +28,6 @@ const Badge: FunctionComponent<IBadgeProps> = ({
 Badge.defaultProps = {
   children: "No content",
   icon: false,
-  link: false,
-  pill: false,
-  round: false,
-  to: "#",
-  type: ""
+  link: false
 };
```

Comment:
Better to add default values for the other optional props like color and type

Reply:
There are no default values for colour and type to make the code more reusable. If we pick a colour or type, the user would have to stick to it. What do you think?